README.md

```diff
@@ -2,8 +2,52 @@
 
 ## Purpose
 
-_... help fill this in!_
+Purpose is *not* to respond to reports of security issues with Node.js, that is
+done by the "security team", see https://github.com/nodejs/node#security
+
+*todo*: should the team be identified in
+https://github.com/nodejs/node#security?
```

Comment:
@sam-github Are you talking about https://github.com/orgs/nodejs/teams/security ? I think most of the teams in nodejs are private (i.e. you can't see who's in them unless you're a member of the nodejs org). I can see who's in that team.

If you meant that the security team list should be duplicated in the README, then fair enough.